mne_bids/mne_bids.py

```diff
@@ -223,10 +245,21 @@ def _participants_tsv(raw, subject_id, group, fname, verbose):
         Name of group participant belongs to.
     fname : str
         Filename to save the participants.tsv to.
+    overwrite : str, one of ('overwrite', 'error', 'clear')
+        How the file should be handled if is exists already.
+        Defaults to 'overwrite'.
+        If overwrite = `overwrite` the previously existing file will have the
+        new data added to it, overwriting any duplicate entries found.
+        If overwrite = `clear` the previously existing file will be removed and
+        replaced with the new data.
```

Comment:
Yeah, this was the problem I had with having the default as `overwrite`, since it actually appends the already existing files. I wasn't sure what else to write though as I think this behaviour is what we want.